public/validation.js

```diff
@@ -0,0 +1,149 @@
+/// add validataion when user enter any youtuber's birthday
+
+// category list of all youtube category
+const categoryList = [
+  "Film and Animation",
+  "Autos and Vehicles",
+  " Music",
+  "Pets & Animals",
+  "Sports",
+  "Travel and Events",
+  "Gaming",
+  "People and Blogs",
+  "Comedy",
+  "Entertainment",
+  "News and Politics",
+  "Education",
+  "Science and Technology",
+  "How-to and Style",
+  "Nonprofits and Activism",
+];
```

Comment:
You can add one more category "Others" and in the music category remove that extra space.